packages/web/src/hooks/useTabs/useTabs.tsx

```diff
@@ -1133,6 +1133,7 @@ const useTabs = ({
     (newIndex: number) => {
       if (isControlled) {
         onChangeComplete(activeIndex, newIndex)
+        onTabClickCb && onTabClickCb(tabs[newIndex].label)
```

Comment:
Cool sup with this?

Reply:
without this line, switching tabs by clicking on them doesn't work because it's in a controlled state via `selectedTabLabel`. i think this use case is slightly more complicated than others because we have to click on tabs to switch and be able to switch tabs based on navigation like clicking on show all profiles.